### DISCUSSION TITLE
How to send mail on behalf of user

### DISCUSSION THREAD

**Discussion by w99910** [emersion/maddy#710] [Q&A]
I have setup maddy mail server at `example.com` and created user called `mail@example.com`. But when sending email using my mail server, I would like to set different sender address such as `support@example.org`. 

I have found [an article](https://postmarkapp.com/support/article/846-how-can-i-send-on-behalf-of-my-users#verifying-email) about sending verification email to sender address.

Please let me know how I can implement it. 

So far, I got error "553 5.7.0 Unauthorized use of sender address". 

Many thanks.

**Comment by taoso** [comment]
maybe you need to try 

```
submission tls://0.0.0.0:465 tcp://0.0.0.0:587 {
    limits {
        # Up to 50 msgs/sec across any amount of SMTP connections.
        all rate 50 1s
    }

    auth &local_authdb

    source $(local_domains) {
        check {
            authorize_sender {
                prepare_email &local_rewrites
            }
        }
...
table.chain local_rewrites {
    optional_step static {
        entry support@example.org mail@example.com
    }
}
```
